examples/radial_interleafed_example.py

```diff
@@ -0,0 +1,125 @@
+NIFTI = False
+
+# %% [markdown]
+
+# # Download example data
+# The example data is taken from (https://zenodo.org/records/10669837).
+# It entails data acquired using radial-interleafed sequences written in pulseq.
+# %%
+# Create a temporary directory to save the downloaded files
+data_folder = Path(tempfile.mkdtemp())
+print(f'Data will be saved to: {data_folder}')
+
+# Initialize attempt counter and set maximum number of attempts
+attempt = 1
+max_attempts = 5
+
+# Start the download process
+while attempt <= max_attempts:
+    # Zenodo command for downloading
+    zenodo_cmd = f'zenodo_get 10.5281/zenodo.10664974 -o {str(data_folder)}'
+
+    # Execute the Zenodo command
+    out = subprocess.call(zenodo_cmd, shell=True)
+
+    # Check if the download was successful
+    if out == 0:
+        print('Download successful.')
+        break
+    else:
+        print(f'Download failed! Attempt {attempt} of {max_attempts}')
+        attempt += 1
+
+if attempt > max_attempts:
+    # Abort after reaching the maximum number of attempts
+    raise ConnectionError('Zenodo download failed after 3 attempts!')
+
+# %%
+filepath = data_folder
+seq_filename = '20240130_CEST_interleafed_radial_256px_fov256_8mm_50spokes_100ms.seq'
+h5_filename = 'meas_MID00044_FID00370_20240130_CEST_interleafed_radial_256px_fov256_8mm_50spokes_100m.h5'
+# %% [markdown]
+# # Read in data
+# Data is read in using the downloaded .h5 file and a corresponding
+# trajectory calculator (cartesian in this case). Given the trajectory
+# calculator, mrpro will automatically read, sort and validate the data.
+# %%
+data = KData.from_file(
+    ktrajectory=KTrajectoryRadial2D(),
+    filename=f'{filepath}/{h5_filename}',
+)
+# %% [markdown]
+# # Reconstruction
+# Given the loaded data, the Fourier Operator is initialized next,
+# with which the appropriate Fourier Transform is carried out.
+# Furthermore, density compensation and coil sensitivity maps are created to
+# control for varying coil sensitivities ad densities throughout the scan.
+# %%
+# Densitiy compensation
+dcf = DcfData.from_traj_voronoi(traj=data.traj)
+
+# %%
+# perform FT and CSM
+ft_op = FourierOp(
+    recon_shape=SpatialDimension(1, 256, 256),
+    encoding_shape=SpatialDimension(1, 256, 256),
+    traj=data.traj,
+    oversampling=SpatialDimension(1, 1, 1),
+)
+(xcoils,) = ft_op.H(data.data * dcf.data)
+idata = IData.from_tensor_and_kheader(xcoils, data.header)
+
+smoothing_width = SpatialDimension(z=1, y=5, x=5)
+csm = CsmData.from_idata_walsh(idata, smoothing_width)
+sensitivity_op = SensitivityOp(csm)
+(x,) = sensitivity_op.H(xcoils)
+
+# %% [markdown]
+# # See the results
+# %%
+image = x.abs().square().sum(1).sqrt()
+image = image.squeeze()
+for i in image:
+    print(i.shape)
+    plt.matshow(torch.abs(i))
+    break
+# %% [markdown]
+# # Save reconstructed images
+# In case you would like to save the reconstructed images,
+# set the NIFTI flag at the top to True
+# %%
+if NIFTI:
```

Comment:
Q: is there a reason for this flag? why not just always do it in the example?

Reply:
i see,
i wouldn't mind removing this completely from the examples, as this is not part of mrpro (yet)

Reply:
Sounds good, I will remove it